app/views/merchant/dashboard/index.html.erb

```diff
@@ -9,4 +9,16 @@
   <p><%= @merchant.zip %></p>
 
 
-  </section>
+</section>
+
+<% @pending_data.each do |pending_datum|%>
+  <% pending_datum.flatten.each do |info|%>
```

Comment:
Again - great job for red to green!

Reply:
Thanks!  There's definitely a more eloquent way, but we can refactor to get to that.